README.md

```diff
@@ -11,44 +11,50 @@ The goal of this project is to build project with following tools:
  * Build system: [`CMake`](https://cmake.org/)
  * C++ compiler: `g++`
  * Libraries: `STL` only
- * Code coverage: [`lcov`](http://ltp.sourceforge.net/coverage/lcov.php) (note: it should show the code coverage is below 100%)
+ * Code coverage report: [`lcov`](http://ltp.sourceforge.net/coverage/lcov.php) (note: it should show the code coverage is below 100%)
  * [`CodeCov`](https://codecov.io/) (code coverage is measured by CodeCov).
  * Source: multiple files
 
 ## Special Thanks
 Goes to [Richel Bilderbeek](https://github.com/richelbilderbeek) for inspiration and all work on [Travis CI tutorials](https://github.com/richelbilderbeek/travis_cpp_tutorial). 
-Here is [link](https://github.com/richelbilderbeek/travis_cmake_gcc_cpp11) to the same project with structure (except no `lcov`) and here is [list](https://github.com/richelbilderbeek/travis_cpp_tutorial/blob/master/statuses.md) of all his projects if you want make build with travis but with different configuration.
+Here is a [link](https://github.com/richelbilderbeek/travis_cmake_gcc_cpp11) to a project with the same structure (without `lcov`),
+and here is a [list](https://github.com/richelbilderbeek/travis_cpp_tutorial/blob/master/statuses.md) of all his Travis configuration examples.
 
 ## Prerequisites
-
-To build the project you need to install `CMake`. [Here](https://cmake.org/install/) are the instructions. To create code coverage report you need to install `lcov`. [`Download lcov`](http://ltp.sourceforge.net/coverage/lcov.php) from here you can download latest `lcov` and here are [`instructions`](http://ltp.sourceforge.net/coverage/lcov/readme.php). This reports will be later uploaded to CodeCov servers.
+To build the project you need to install `CMake`. [Install instructions](https://cmake.org/install/)
+To display a code coverage report in the console, install `lcov`. [`Download lcov`](http://ltp.sourceforge.net/coverage/lcov.php) from here you can download latest `lcov` and here are [`instructions`](http://ltp.sourceforge.net/coverage/lcov/readme.php).
+(These reports will also be uploaded to CodeCov servers.)
 
 ## Guide
-### Travis Setup
+* Compile with code coverage instrumentation enabled [(GCC)](https://gcc.gnu.org/onlinedocs/gcc/Instrumentation-Options.html).
+* Execute the tests to generate the coverage data.
+* (Optionally) generate and customize reports with `lcov`.
```

Comment:
Again this is duplicate and wrong. If you don't use `-f` for the codecov script it will not honor your customization and collect it again

Reply:
Using `-f` and added a description on usage without lcov.